lib/AutoDragSortableView.js

```diff
@@ -106,6 +108,7 @@ export default class AutoDragSortableView extends Component{
     componentDidMount() {
         this.initTag()
         this.autoMeasureHeight()
+        this.isHasMeasure = true;
```

Comment:
Prevent autoMeasureHeight from running again due to componentDidUpdate(). If we don't do this then before the first reorder (which sets isHasMeasure to true) pull to refresh acts very janky because we are pulling to refresh so the refreshing prop updates but then autoMeasureHeight causes the ScrollView to scroll.